src/explore-education-statistics-common/src/modules/table-tool/components/FixedMultiHeaderDataTable.tsx

```diff
@@ -133,10 +133,12 @@ const FixedMultiHeaderDataTable = forwardRef<HTMLElement, Props>(
           />
         </div>
 
-        <FigureFootnotes
-          footnotes={footnotes}
-          headingHiddenText={footnotesHeadingHiddenText}
-        />
+        <div className="govuk-!-width-two-thirds">
```

Comment:
Wouldn't this mean that the footnotes will always be 2/3 width? Wouldn't this cause data block footnotes to be too narrow? To me, it seems like it'd more appropriate for these to be full width.

Reply:
Doh yes, I've changed it now.